appinventor/appengine/src/com/google/appinventor/client/youngandroid/YoungAndroidFormUpgrader.java

```diff
@@ -1567,9 +1567,9 @@ private static int upgradeSpeechRecognizerProperties(Map<String, JSONValue> comp
 
   private static int upgradeSpreadsheetProperties(Map<String, JSONValue> componentProperties,
       int srcCompVersion) {
-    if (srcCompVersion < 2) {
+    if (srcCompVersion < 3) {
       // Various methods were renamed in the blocks editor.
-      srcCompVersion = 2;
+      srcCompVersion = 3;
```

Comment:
Usually we keep a branch for each version, even if nothing has changed. See some of the other functions as examples. The compiler will optimize away the extra branching.